src/database/sqlite/sqlite_metadata.c

```diff
@@ -668,6 +668,16 @@ int sql_init_meta_database(db_check_action_type_t rebuild, int memory)
             if (rebuild & DB_CHECK_RECOVER)
                 return 0;
         }
+
+        snprintfz(sqlite_database, sizeof(sqlite_database) - 1, "%s/.netdata-meta.db.delete", netdata_configured_cache_dir);
+        rc = unlink(sqlite_database);
```

Comment:
Should we log possible errors? It is not a problem if it is behind a flag.

Reply:
This should give an error because it is ok that the file does not exist (means no action should be taken)

However, I will add a check and log an error if the following rename fails